pkgs/os-specific/linux/evdi/default.nix

```diff
@@ -16,7 +16,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-em3Y56saB7K3Wr31Y0boc38xGb57gdveN0Cstgy8y20=";
   };
 
-  env.NIX_CFLAGS_COMPILE = "-Wno-error -Wno-error=sign-compare";
+  env.NIX_CFLAGS_COMPILE = toString [
+    "-Wno-error"
+    "-Wno-error=discarded-qualifiers" # for Linux 4.19 compatibility
+    "-Wno-error=sign-compare"
+  ];
```

Comment:
This seems like an unrelated change?

Reply:
Thanks for the review @alyssais 
Indeed I went too fast, I splitted this ZHF failure for Linux 4.19 in another commit.
Is it okay for you?

Reply:
That's much better, thanks.